tools/pythonpkg/src/include/duckdb_python/pandas_type.hpp

```diff
@@ -17,6 +17,18 @@ class DataFrame : public py::object {
 	}
 };
 
+class PolarsDataFrame : public py::object {
+public:
+	PolarsDataFrame(const py::object &o) : py::object(o, borrowed_t {}) {
+	}
+	using py::object::object;
+
+public:
+	static bool check_(const py::handle &object) {
+		return !object.is_none();
```

Comment:
This one is my bad - it should import the class and call isinstance with it. Should probably fix the pandas one too